src/main/resources/templates/myplaces.html

```diff
@@ -59,11 +59,7 @@
     </style>
 </head>
 <body>
-<header>
-    <a href="#">My Places</a>
-    <a href="#">Search</a>
-    <a href="#">Settings</a>
-</header>
+<div th:replace="~{fragments/navbar :: navbar}"></div>
```

Comment:
I chose to include Thymeleaf on each of our pages along with your header, that way Thymeleaf can be used to pass in data from your controller or use fragments like the one shown here.

Source: https://martinfowler.com/bliki/TechnicalDebt.html